src/core/Form/RadioButton/RadioButtonGroup.tsx

```diff
@@ -1,23 +1,152 @@
-import React, { Component } from 'react';
+import React, { Component, ReactNode } from 'react';
 import { default as styled } from 'styled-components';
 import { TokensProp, InternalTokensProp } from '../../theme';
-import {
-  RadioButtonGroup as CompRadioButtonGroup,
-  RadioButtonGroupProps as CompRadioButtonGroupProps,
-} from '../../../components/Form/RadioButtonGroup';
+import { HtmlDiv, HtmlSpan, HtmlFieldSet, HtmlLegend } from '../../../reset';
+import { VisuallyHidden } from '../../../components/Visually-hidden/Visually-hidden';
+import { RadioButtonProps } from './RadioButton';
 import { baseStyles } from './RadioButtonGroup.baseStyles';
 import { withSuomifiDefaultProps } from '../../theme/utils';
+import { AutoId } from '../../../utils/AutoId';
 import classnames from 'classnames';
 
 const baseClassName = 'fi-radio-button-group';
+const radioButtonGroupClassNames = {
+  container: `${baseClassName}_container`,
+  label: `${baseClassName}_label`,
+  hintText: `${baseClassName}_hintText`,
+};
 
-export interface RadioButtonGroupProps
-  extends CompRadioButtonGroupProps,
-    TokensProp {}
+type Label = 'hidden' | 'visible';
+
+export interface RadioButtonGroupProps extends TokensProp {
+  /** Custom classname to extend or customize */ className?: string;
+  /** RadioButton or ReactNode */
+  children: Array<React.ReactElement<RadioButtonProps> | ReactNode>;
+  /** Hint text to be displayed under the label. */
+  hintText?: string;
+  /** Label for the group */
+  label: string;
```

Comment:
Should the `label` prop to be `labelText` instead to be more consistent with other components?

Reply:
Would be a breaking change, but I guess we could change it since there are already other breaking changes.

Reply:
Changed